Taskfile.yml

```diff
@@ -14,10 +14,12 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+#
 version: "3"
 
 vars:
-  BASEIMG: ghcr.io/nuvolaris/openserverless-streamer
+  BASETAG: 0.1.0-incubating
```

Comment:
can you make this an RC tag? The ASF doesn't allow you to push tags on Docker images that look like release tags - unless the tag is an associated with a voted on release.

Reply:
you mean something like 0.1.0-rc0-incubating? I understood that incubating must always be contained in the TAG.